app/controllers/cohorts/client_notes_controller.rb

```diff
@@ -31,7 +31,7 @@ def create
       @cohort_client.touch
       # send notifications
       if note_params[:send_notification].present? && note_params[:recipients].present?
-        token = Token.tokenize(cohort_cohort_client_client_notes_path(client_id: @cohort_client.client_id))
+        token = Token.tokenize(client_notes_path(client_id: @cohort_client.client_id))
```

Comment:
to confirm, this is a note added on a cohort, but the link is going to the client/notes page.  (I believe that's the intended behavior)

Reply:
Correct -- this is controller that creates a note that is visible on the client's Notes tab with the label "Cohort note", and this send the user to the tab instead of the modal.